consensus/checks.go

```diff
@@ -11,6 +11,11 @@ import (
 const MaxBlockNumDiff = 100
 
 func (consensus *Consensus) validatorSanityChecks(msg *msg_pb.Message) bool {
+	consensus.getLogger().Info().
```

Comment:
is it for debug only? if yes, we can use Debug().

Reply:
done.